cherry_picker/cherry_picker.py

```diff
@@ -794,8 +794,12 @@ def cherry_pick_cli(
 
     click.echo("\U0001F40D \U0001F352 \u26CF")
 
-    chosen_config_path, config = load_config(config_path)
-
+    try:
+        chosen_config_path, config = load_config(config_path)
+    except ValueError as exc:
+        click.echo("You're not inside a Git tree right now! \U0001F645", err=True)
```

Comment:
I think this `ValueError` is expected to be raised by `get_sha1_from` when executing a git command and an error occurs. But the error may not be caused by the fact that the pwd is not in a git tree; it could be some other errors, like git not being installed.

I suggest just echoing some words like "Got an error when executing git command," and echoing the stderr output from the git command.